driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -411,11 +442,17 @@ void onSet(
 
     private volatile Connection.ResponseHandler connectionHandler;
 
+    private final TracingInfo tracingInfo;
+    private TracingInfo currentTracingInfo;
```

Comment:
`parentTracingInfo` and `currentChildTracingInfo` would better describe the relationship between those tracing infos.
```suggestion
    private TracingInfo currentChildTracingInfo;
```